LiskHQ/lisk-desktop
# Delegate search points to transaction tab - Closes #4023

### What was the problem?

This PR resolves #4023

### How was it solved?

This issue was solved by adding the `tab` query param to the accounts route when a delegate search item was clicked

### How was it tested?

- Search for a delegate
- Click on any of the delegates that appear in the suggestion list
- you should be routed to the delegate profile page not delegate transactions
